examples/alert.json

```diff
@@ -2,23 +2,29 @@
 	"apiVersion": "synopsys.com/v1",
 	"kind": "Alert",
 	"metadata": {
-	  "clusterName": "",
-	  "name": "alert-test"
+		"clusterName": "",
+		"name": "alert-test"
 	},
 	"spec": {
-	  "namespace": "alert-test",
-	  "registry": "docker.io",
-	  "imagePath": "blackducksoftware",
-	  "alertImageName": "blackduck-alert",
-	  "alertImageVersion": "2.1.0",
-	  "cfsslImageName": "hub-cfssl",
-	  "cfsslImageVersion": "4.8.1",
-	  "blackduckHost": "<<HUB_HOST>>",
-	  "blackduckUser": "sysadmin",
-	  "blackduckPort": 443,
-	  "port": 8443,
-	  "standAlone": true,
-	  "alertMemory": "512M",
-	  "cfsslMemory": "640M"
+		"namespace": "alert-test",
+		"registry": "docker.io",
+		"imagePath": "blackducksoftware",
+		"alertImageName": "blackduck-alert",
+		"alertImageVersion": "3.1.0",
+		"cfsslImageName": "blackduck-cfssl",
+		"cfsslImageVersion": "1.0.0",
+		"port": 8443,
+		"standAlone": true,
+		"alertMemory": "2560M",
+		"cfsslMemory": "640M",
+		"pvcMemory": "100M",
```

Comment:
This should be more than 100M

Reply:
Paulo mentioned the dir-alert volume needing 5GB of space so I can set that pvcMemory field to "5G"